src/components/Summary/NoFeeSummary.vue

```diff
@@ -37,7 +39,7 @@
     </template>
 
     <!-- Folio Number -->
-    <template>
+    <template id="no-fee-folio">
```

Comment:
What this id used for?

Reply:
Nothing yet, but will be for unit testing once the folio piece is done. Just put it in now, so it followed the same id pattern as the other 😁